sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormRecognizerClient/RecognizeCustomFormsLiveTests.cs

```diff
@@ -231,6 +232,7 @@ public async Task StartRecognizeCustomFormsWithLabelsCanParseMultipageForm(bool
         }
 
         [RecordedTest]
+        [Ignore("https://github.com/Azure/azure-sdk-for-net/issues/28556")]
         public async Task StartRecognizeCustomFormsWithLabelsCanParseBlankPage()
```

Comment:
Blank pages have been making the service fail for some time now. Still need to investigate what errors we actually get.

Reply:
I wonder how much investigation needs to happen in our side if this code hasn't touched and the service also stop changes in v2.1